official/resnet/resnet_run_loop.py

```diff
@@ -384,16 +405,37 @@ def input_fn_eval():
       benchmark_logger = logger.BenchmarkLogger(flags.benchmark_log_dir)
       benchmark_logger.log_estimator_evaluation_result(eval_results)
 
+  # Return the classifier after all epochs are completed.
+  return classifier
+
+
+def export_savedmodel(classifier, export_dir, shape, batch_size):
+  """Exports a saved model for the given classifier."""
+  input_receiver_fn = export.build_tensor_serving_input_receiver_fn(
+      shape, batch_size=batch_size)
+  classifier.export_savedmodel(export_dir, input_receiver_fn)
+  return classifier
+
+
+def warn_on_multi_gpu_export(multi_gpu=False):
```

Comment:
If user export the model on a single GPU mode and try to run the exported model on a multi GPU env, will it work?

Reply:
Probably not. Although serving in multi-GPU mode is not really a thing that's typically done, I think; parallelization is handled more efficiently on the request-partitioning side, I believe, rather than inside the model itself, since there is no need to update shared weights/other info.